Makefile

```diff
@@ -1,6 +1,16 @@
+install:
+	pip install -r requirements.txt       # pulls in CDKv2 `aws-cdk-lib` and `construct` libs
+	npm install -g aws-cdk@2.15.0
+	
+	brew tap aws/tap
+	brew reinstall aws-sam-cli
 build:
-	sam-beta-cdk build
+#   Build lambda Layer
+	for dir in $(shell find ./lambdas/layers -maxdepth 1 -mindepth 1 -type d -exec basename {} \;); do ./lambdas/layers/create_layer_package.sh $$dir; done
+#   Build Cdk to cdk.out
+	cdk synth
 deploy: build
-	cdk deploy -a .aws-sam/build --profile ${AWS_PROFILE}
+	cdk deploy UmccrEventBus --profile ${AWS_PROFILE}
+	cdk deploy UmccrEventBusSchemaStack --profile ${AWS_PROFILE}
```

Comment:
What's the reason for `--profile ${AWS_PROFILE}`?

Do we need/want to restrict credential resolution options?

Reply:
Yes could removed. Was thinking of allowing to deploy on a certain profile (allowing to choose for `prod` or `dev`) when necessary. But I guess could be solve when by default setting it on environment variable. 

Good to remove, specially when building on codebuild it might not have `AWS_PROFILE` as their environment variable